app/routes/docs/about/history.mdx

```diff
@@ -0,0 +1,48 @@
+---
+handle:
+  breadcrumb: History
+---
+
+import history from '~/img/gcn_history.png'
```

Comment:
Can we update the GCN history arrow, by including General coordinate name, also GCN classic over Kafka notices and circulars transition? I did a little change for AAS poster.
<img width="274" alt="GCN_History" src="https://github.com/nasa-gcn/gcn.nasa.gov/assets/103223246/a82dab91-c11f-450d-a9c3-5f8a626e2917">

Reply:
That's a great idea, but the source image is in https://github.com/nasa-gcn/gcn-presentation. @Vidushi-GitHub, would you please open an issue there?

Reply:
I asked Britt for an updated version.